### PR TITLE
Have `cut_release` handle "config/default" and generate cops doc.

### PR DESCRIPTION
**Pull request by marcandre**
* task `cut_release` will replace the string "<<next_version>>" in the default.yml file with the version being cut (matching is permissive: quoted or unquoted, with "-" or "_", etc.)
* task `generate_cops_documentation` now does nothing and only prints out that doc generation is now automatic
* processing code moved to new task `update_cops_documentation` (no public description) that is triggered by `cut_release`
* new cop generator uses this by default instead of the next minor version

Did I forget anything?

### CODE REVIEW

**Comment by bbatsov** [comment]
> <<next_version>>

I think we can shorten this to `<<next>>`. That would simply the matching a regexp a bit. 

> Did I forget anything?

I don't think so.

**Comment by marcandre** [comment]
Changed to `<<next>>` 👍
